### DISCUSSION TITLE
`minemacs-disabled-packages` is not respected

### DISCUSSION THREAD

**Discussion by azzamsa** [abougouffa/minemacs#87] [Q&A]
Hi.

I have set `(push 'tempel minemacs-disabled-packages)` but, the tempel function is still there.

![image](https://github.com/abougouffa/minemacs/assets/17734314/7ebf8e0b-7200-4eef-ab8a-d75948ba3887)

I have restarted emacs multiple times, and run `straight-remove-unused-repos`.

Thanks. :heart:

**Comment by abougouffa** [comment]
It seems that `tempel` is installed as `tempel-collection` dependency in your case. The `minemacs-disabled-packages` ensures only that the package don't get explicitly installed, however, if another package depends on it, it will get installed.

TLDR; add this:

```elisp
(push 'tempel-collection minemacs-disabled-packages)
```

**Reply by azzamsa** [reply]
I think we should put an example of multiple disabled packages. Is it possible to write in one line? `(push 'foo minemacs-disabled-packages)` or `(push '(foo bar) minemacs-disabled-packages)`.

**Reply by abougouffa** [reply]
Good idea, I will add support for adding several packages as a list.
